.github/workflows/e2e-maestro-template.yml

```
@@ -454,20 +456,65 @@ jobs:
           github-token: ${{ github.token }}
           gh-job-name: ${{ inputs.tsio-shard-name != '' && inputs.tsio-shard-name || 'Maestro E2E Tests (iOS)' }}
           # Action input is named json-path; Maestro uploads JUnit XML bytes.
           json-path: build/maestro-report.xml
           screenshots-dir: build/maestro-artifacts-tsio
 
+      - name: Post per-job TSIO commit status
+        if: always() && inputs.tsio-config != '' && vars.TSIO_UPLOAD_ENABLED != 'false' && steps.tsio-cfg.outputs.config != ''
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+          TSIO_CONFIG: ${{ steps.tsio-cfg.outputs.config }}
+          MATTERMOST_E2E_WEBHOOK_URL: ${{ secrets.MM_MOBILE_E2E_WEBHOOK_URL }}
```

actionlint (1.7.12)

[error] property "mm_mobile_e2e_webhook_url" is not defined in object type {actions_runner_debug: string; actions_step_debug: string; github_token: string; mm_mobile_detox_aws_access_key_id: string; mm_mobile_detox_aws_secret_access_key: string; mm_mobile_e2e_admin_email: string; mm_mobile_e2e_admin_password: string; mm_mobile_e2e_admin_username: string; mm_mobile_e2e_zephyr_api_key: string}

(expression)